linux-anvil2/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM ppc64le/centos:7
+
+MAINTAINER conda-forge <conda-forge@googlegroups.com>
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Add a timestamp for the build. Also, bust the cache.
+ADD http://tycho.usno.navy.mil/timer.html /opt/docker/etc/timestamp
```

Comment:
Feel free to just drop these two lines. It was needed to bust the cache on Quay.io, but isn't necessary if we use DockerHub, build on CI without the image, or if we build with the `--no-cache` option on CI and push.

Reply:
Should add these lines are causing the CI failures for this PR as it appears USNO made some changes to their server for HTTPS (was HTTP only before), but their certificate isn't accepted as valid.

Reply:
Done